adserver/tasks.py

```diff
@@ -83,32 +84,83 @@ def _default_filters(impression_type, start_date, end_date):
 
 
 @app.task()
-def daily_update_geos(day=None):
+def daily_update_geos(day=None, geo=True, region=True):
```

Comment:
I realize there's some common code, but combining this feels awkward and hard to test. I wouldn't mind just seeing two tasks.

Reply:
Pretty soon the geos part will go away, I think. I just didn't want to run the exact same query twice, but I can split it up if we feel strongly. I think the real path forward is deleting the old one after we're happy w/ the new reporting.

Reply:
I agree with deleting the geo report stuff afterwards but I feel like it would be cleaner then to have a separate function that we just blow away rather than refactoring this to work 2 ways and then un-factoring it.

Reply:
Yea, definitely cleaner, but uses more DB resources. I was opting to not hammer the DB as hard, but it's definitely a tradeoff.